gwpy/timeseries/tests/test_timeseries.py

```diff
@@ -842,6 +842,40 @@ def test_demodulate(self):
         assert ph.unit == 'rad'
```

Comment:
@mattpitkin, because of the switch to 32-bit floats, there is some loss in precision when calculating the phase of a demodulated signal. I think the loss is acceptable, but the unit tests for `TimeSeries.demodulate` will have to be updated.

With apologies for the extra hassle, can you make the following changes in `test_timeseries.py::TestTimeSeries::test_demodulate`?

* Line 830: change `rtol` to `1e-2`
* Line 838: change `rtol` to `1e-2`
* Line 843: change `rtol` to `1e-2`

Note, if you have the `pytest` package installed then you can run this unit test on the command-line to check that it behaves the way you want:

```bash
$ python -m pytest gwpy/timeseries/tests/test_timeseries.py::TestTimeSeries::test_demodulate
```